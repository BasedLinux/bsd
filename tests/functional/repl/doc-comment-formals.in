:l doc-comments.bsd
"Note that this is not yet complete"
:doc documentedFormals
