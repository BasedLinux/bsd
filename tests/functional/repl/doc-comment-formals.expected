Bsd <bsd version>
Type :? for help.

bsd-repl> :l doc-comments.nix
Added <number omitted> variables.

bsd-repl> "Note that this is not yet complete"
"Note that this is not yet complete"

bsd-repl> :doc documentedFormals
Function `documentedFormals`\
  â€¦ defined at /path/to/tests/functional/repl/doc-comments.nix:104:5


Finds x
